chat-plugins/trivia.js

```diff
@@ -1258,43 +1251,44 @@ const commands = {
 		}
 
 		let score = triviaData.leaderboard[userid];
-		if (!score) return this.sendReplyBox("User '" + name + "' has not played any trivia games yet.");
+		if (!score) return this.sendReplyBox(`User '${name}' has not played any trivia games yet.`);
 
 		this.sendReplyBox(
-			"User: <strong>" + name + "</strong><br />" +
-			"Leaderboard score: <strong>" + score[0] + "</strong> (#" + score[3] + ")<br />" +
-			"Total game points: <strong>" + score[1] + "</strong> (#" + score[4] + ")<br />" +
-			"Total correct answers: <strong>" + score[2] + "</strong> (#" + score[5] + ")" +
-			(triviaData.ugm ? "<br />UGM points: <strong>" + triviaData.ugm[userid] + "</strong>" : "")
+			`User: <strong>${name}</strong><br />`	 +
+			`Leaderboard score: <strong>${score[0]}</strong> (#${score[3]})<br />` +
+			`Total game points: <strong>${score[1]}</strong> (#${score[4]})<br />` +
+			`Total correct answers: <strong>${score[2]}</strong> (#${score[5]})` +
+			(triviaData.ugm ? `<br />UGM points: <strong>${triviaData.ugm[userid]}</strong>` : "")
 		);
 	},
 	rankhelp: ["/trivia rank [username] - View the rank of the specified user. If none is given, view your own."],
 
 	ladder: function (target, room) {
 		if (room.id !== 'trivia') return this.errorReply('This command can only be used in Trivia.');
 		if (!this.runBroadcast()) return false;
-
 		let ladder = triviaData.ladder;
 		let leaderboard = triviaData.leaderboard;
 		if (!ladder.length) return this.errorReply("No trivia games have been played yet.");
 
-		let buffer = "|raw|<div class=\"ladder\"><table>" +
+		let buffer = "|raw|<div class=\"ladder\" style=\"overflow-y: scroll; max-height: 300px;\"><table style=\"width: 100%\">" +
```

Comment:
You don't need to replace all of them, but at least replace the ones you're touching.